test/e2e/utils_test.go

```diff
@@ -503,3 +598,84 @@ func ensureCRPAndRelatedResourcesDeletion(crpName string, memberClusters []*fram
 	// Delete the created resources.
 	cleanupWorkResources()
 }
+
+// verifyWorkPropagationAndMarkAsApplied verifies that works derived from a specific CPR have been created
+// for a specific cluster, and marks these works in the specific member cluster's
+// reserved namespace as applied.
+//
+// This is mostly used for simulating member agents for virtual clusters.
```

Comment:
```suggestion
// This is mostly used for simulating member agents for fake clusters.
```
nit

Reply:
In actuality all of the clusters we use are fake

Reply:
ah, i remember we do create 3 kind clusters and install the agents there. is my understanding correct?